lib/iris/tests/test_coordsystem.py

```diff
@@ -322,5 +325,57 @@ def test_south_cutoff(self):
         self.assertEqual(ccrs.cutoff, 30)
 
 
+class Test_Mercator_construction(tests.IrisTest):
+    def test_merc(self):
+        tm = merc()
+        self.assertXMLElement(tm, ("coord_systems", "Mercator.xml"))
+
+
+class Test_Mercator_repr(tests.IrisTest):
+    def test_merc(self):
+        tm = merc()
+        expected = "Mercator(longitude_of_projection_origin=90.0, "\
+                   "ellipsoid=GeogCS(semi_major_axis=6377563.396, semi_minor_axis=6356256.909))"
```

Comment:
This line goes over the 80 character limit. I assume that this was not picked up by the coding standards test because this is one of the files skipped by the test. Nevertheless as this is new code it would be good if it obeyed the 80 character limit!